pom.xml

```diff
@@ -76,6 +92,34 @@
             <version>${hamcrest-all-version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-junit4</artifactId>
+            <version>${powermock.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
```

Comment:
Doesn't look like this dependency so can be removed but we should try not to use powermock as in the past we have had to strip it from our services https://companieshouse.atlassian.net/wiki/spaces/DEV/pages/936345626/Removing+Powermock

Reply:
I can't remember why I added it in. I've removed it now.